runtime/CMakeLists.txt

```diff
@@ -7,6 +7,6 @@ set(CMAKE_CXX_STANDARD 20)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 add_subdirectory(../engine engine_dir)
-add_executable(spear-application main.cc)
+add_executable(main.cc)
```

Comment:
This seems very cursed, but it just might be a skill issue from me. If the binary name is the same this passes.